source/slang/slang-parameter-binding.cpp

```diff
@@ -585,7 +588,7 @@ static bool findLayoutArg(
         if( modifier )
         {
             *outVal = (UInt) strtoull(String(modifier->valToken.Content).getBuffer(), nullptr, 10);
-            return true;
+return true;
```

Comment:
tabbing problem?

Reply:
Looks like it. My editor seems to have been doing some weird things where it is changing indentation on lines unrelated to my edits.